### DISCUSSION TITLE
Seeding and streamlines

### DISCUSSION THREAD

**Discussion by staunton-golding** [dipy/dipy#3167] [Tractography / Tractometry] [answered]
Hello,

I surface seeded an image with 2 million seeds, then generated streamlines from those seeds with deterministic maximum direction getter and local tracking. This though results in 2.7 million streamlines. How can there be more streamlines than seeds??? I am struggling to make sense of this.

Best,
Hank

**Comment by Garyfallidis** [comment, marked as answer]
Multiple tracks can be generated by one seed. For example, if crossings are there.

**Reply by staunton-golding** [reply]
Sweet, thank you!
